Source/WebKit/UIProcess/WebPageProxy.cpp

```diff
@@ -8989,8 +8989,10 @@ void WebPageProxy::setMediaVolume(float volume)
     
     if (!hasRunningProcess())
         return;
-    
-    send(Messages::WebPage::SetMediaVolume(volume));
+
+    forEachWebContentProcess([&](auto& webProcess, auto pageID) {
```

Comment:
Honest question: why use by-reference capture default instead of simple by-copy capture of the variables used?